main.js

```diff
@@ -11,3 +11,140 @@ function closemenu() {
 //  binding the event to HTML
 menu.addEventListener('click', openmenu);
 menu.addEventListener('click', closemenu);
+
+const workInfo = [
+  {
+    Id: '1',
+    name: 'Tonic',
+    img: 'snapshoot-portfolio.png',
+    descriptions: 'hfjfjhfconsectetur adipisicing elit. Animi quae quos dolor numquam, quibusdam tempora illum nulla facere sunt deleniti?',
+    technologies: ['HTMl', 'CSS', 'JavaScript'],
+    liveServer: 'See Live',
```

Comment:
- [ optional] ensure when the mobile popup comes up, the portfolio behind is blurry so the user can concentrate on the project before him